src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBER_OF_FIGURES = 4;
```

Comment:
```suggestion
    private static final int NUMBER_OF_FIGURES = 5;
```

Reply:
But zero also number. We have 5 figures at numbers 0, 1, 2, 3, 4